pkg/daemon/update.go

```diff
@@ -1572,6 +1592,40 @@ func (dn *Daemon) atomicallyWriteSSHKey(keys string) error {
 	return nil
 }
 
+// Set a given PasswdUser's Password Hash
+func (dn *Daemon) SetPasswordHash(newUsers []ign3types.PasswdUser) error {
+	// confirm that user exits
+	if len(newUsers) == 0 {
+		return nil
+	}
+
+	// mock function to mock the nodes existence  (it shouldnt run if no node exists)
```

Comment:
nit: this comment can be removed as we are not doing any mock while performing passwd update